addons/l10n_se/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    orgnumber = fields.Char('Organisational Number', index=True)
```

Comment:
@daniel-stenlov Hello
I think you add "Organisational Number" for a show in form view and search by it, right?

Because of vat already have Organisational Number so I think we do this in another way if you agree with my upper sentence.

Reply:
Yes, it's used for reports, views and searches. It's also used for external connection with look up services in Sweden. VAT is based on Organisational Number in Sweden, so is also EORI.